src/EVM/Types.hs

```diff
@@ -1233,6 +1233,8 @@ abiKeccak =
 
 -- Utils -------------------------------------------------------------------------------------------
 
+internalError:: [Char] -> [Char]
+internalError a = "internalError " ++ a
```

Comment:
Maybe could stylise `internalError` as `Internal error: ` for better readability since it is an output message.

Reply:
Good point!